assets/helpers/user/userActions.js

```diff
@@ -39,6 +40,10 @@ export function setFullName(name: string): Action {
   return { type: 'SET_FULL_NAME', name };
 }
 
+export function setIsSignedIn(isSignedIn: boolean): Action {
```

Comment:
If it is possible, please add a test for this new action.

Reply:
done